overlord/ifacestate/helpers.go

```diff
@@ -522,6 +527,108 @@ func (c *autoConnectChecker) check(plug *interfaces.ConnectedPlug, slot *interfa
 	return ic.CheckAutoConnect() == nil, nil
 }
 
+func filterUbuntuCoreSlots(candidates []*snap.SlotInfo) []*snap.SlotInfo {
```

Comment:
Perhaps part of the original comment about core transition and preferring "core" slots should be moved here.

Reply:
adding a comment here